Pod/Classes/SEGAdjustIntegration.m

```diff
@@ -29,11 +29,10 @@ - (instancetype)initWithSettings:(NSDictionary *)settings withAnalytics:(SEGAnal
         if ([self trackAttributionData]) {
             [adjustConfig setDelegate:self];
         }
-
         double delayTime = [settings[@"delayTime"] doubleValue];
         // The maximum delay start time of the adjust SDK is 10 seconds.
-        if ([self setDelay] && (delayTime < 10.00)) {
-            [adjustConfig setDelayStart:delayTime];
+        if ([self setDelay]) {
+            [adjustConfig setDelayStart:MIN(delayTime, 10)];
```

Comment:
`delayTime` should be inside the block. No need to read it's value if `setDelay` is false.